src/Math-ODE/PMAB4Stepper.class.st

```diff
@@ -15,11 +15,18 @@ PMAB4Stepper class >> order [
 ]
 
 { #category : #stepping }
-PMAB4Stepper >> doStep3State: thirdState secondState:secondState firstState: firstState initState: initState initTime: initTime [
-
-	self stepSize isNil
-		ifTrue: [ self error: 'step size required by stepper' ].
-	 ^ self stepSize * (  ( (55/24) * (system state: thirdState time:  initTime+ (3* self stepSize) )) - ((59/24)*(system state: secondState time: initTime+ (2* self stepSize))) + ((37/24)*(system state: firstState time: initTime+ self stepSize ) )-   ((3/8)*(system state: initState time: initTime ) )) + thirdState.
+PMAB4Stepper >> doStep3State: thirdState secondState: secondState firstState: firstState initState: initState initTime: initTime [
```

Comment:
I wonder if we should make the signatures (selectors) look similar across the classes. There is a pattern by Sandi Metz, J B Rainsberger and Katrina Owen which involves making similar code look even more similar (i.e. makes the duplication obvious).